src/components/IbulDialog/IdeaOutline.tsx

```diff
@@ -44,7 +51,7 @@ const StyleBottomClosedBtn = styled.button`
   height: 54px;
   width: 360px;
   background-color: ${(props) => props.theme.color.purple700};
-  color: #fff;
+  color: ${(props) => props.theme.color.white};
```

Comment:
```suggestion
  color: ${theme.color.white};
```
prop이 따로 필요 없는 경우들은 이렇게 해도 좋을 것 같아용